src/OSPSuite.Core/Domain/SensitivityAnalyses/SensitivityAnalysisRunResult.cs

```diff
@@ -28,25 +28,25 @@ public IEnumerable<PKParameterSensitivity> AllPKParameterSensitivitiesFor(string
             x => string.Equals(x.QuantityPath, outputPath) && string.Equals(x.PKParameterName, pkParameterName));
       }
 
-      public IEnumerable<OutputParameterSensitivity> AllOutputParameterSensitivitiesFor(string outputPath, string parameterPath)
+      public IEnumerable<OutputParameterSensitivity> AllOutputParameterSensitivitiesFor(string outputPath, string sensitivityParameterName)
       {
          return _allOutputParameterSensitivities.Where(
-            x => string.Equals(x.OutputPath, outputPath) && string.Equals(x.ParameterPath, parameterPath));
+            x => string.Equals(x.OutputPath, outputPath) && string.Equals(x.SensitivityParameterName, sensitivityParameterName));
       }
 
-      public PKParameterSensitivity PKParameterSensitivityFor(string pkParameterName, string outputPath, string parameterName)
+      public PKParameterSensitivity PKParameterSensitivityFor(string pkParameterName, string outputPath, string sensitivityParameterName)
       {
-         return AllPKParameterSensitivitiesFor(pkParameterName, outputPath).Find(x => string.Equals(parameterName, x.ParameterName));
+         return AllPKParameterSensitivitiesFor(pkParameterName, outputPath).Find(x => string.Equals(sensitivityParameterName, x.ParameterName));
       }
 
-      public OutputParameterSensitivity[] OutputParameterSensitivitiesFor(string outputPath, string parameterPath)
+      public OutputParameterSensitivity[] OutputParameterSensitivitiesFor(string outputPath, string sensitivityParameterName)
```

Comment:
using sensitivyParameterName here instead of parmaeterpaht

Reply:
name would be something like "Volume"? or like "Organism|Bone|Volume" (so in fact the path)?

Reply:
no name is the SenstitiveyParameterName (something like Liver-Volume). I am not sure why we are doing it lije this
I would vote to create new method to work with path instead of name everywhere to be honest

Reply:
I prefer path over the name, too. Path is what we define in the first place when starting SA.